Tls

stemi1 opened the pull request:
First merge in github

masmartin committed:
- Supports TLS encrypted connections (Windows client only) when using the referenced fork of the tacopie network module.
- Visual Studio solution and project converted to VS2019

masmartin committed:
- fixed problem evaluating return of ::recv
- clarify return value of ::send

masmartin reviewed (changes requested):
Review mit STM: Some changes requested.

masmartin committed:
changes from review implemented

- Changes implemented
- Fixed a problem in recv_decrypt leading to buffer growth
